src/main/java/com/gluonhq/substrate/model/Triplet.java

```diff
@@ -51,13 +52,8 @@ public static Triplet fromCurrentOS() {
         }
     }
 
-    public Triplet(String arch, String vendor, String os) {
```

Comment:
This should be kept, as third party software can create triplets for any configuration

Reply:
ok